webpack/assets/javascripts/foreman_template_inputs.js

```diff
@@ -1,6 +1,10 @@
 /* eslint-disable func-names */
 import $ from 'jquery';
 
+import store from './react_app/redux';
+
```

Comment:
nit: I see blank lines between imports, in file below there are no blank lines in between, should we just drop them here?

Reply:
felt better at the time of writing. Deleted that one.